docs/advanced/configuration.md

```diff
@@ -39,7 +39,7 @@ These configuration values would be inherited by every run on that system withou
 Move into the chapter example directory:
 
 ```
-cd advanced/configuration
```

Comment:
Ahhh! Sorry, this was my fault. I moved all of these recently to improve the GitPod structure!